docs/faq/faq.md

```diff
@@ -49,6 +49,13 @@ In K3s, there are two types of tokens: `K3S_TOKEN` and `K3S_AGENT_TOKEN`.
 
 If no `K3S_TOKEN` is defined, the first K3s server will generate a random token during initial startup. The result is part of the content in `/var/lib/rancher/k3s/server/token`. For example, `K1070878408e06a827960208f84ed18b65fa10f27864e71a57d9e053c4caff8504b::server:df54383b5659b9280aa1e73e60ef78fc`. The token in this example is `df54383b5659b9280aa1e73e60ef78fc`. The full format with the `K10` prefix includes a hash of the cluster's CA certificate, which can be used to ensure that nodes are joining the correct cluster and are not subject to a man-in-the-middle attack during the join process. This full token cannot be generated prior to initial cluster startup, before the cluster CA has been generated.
 
+
+### How compatible are different versions of K3s?
```

Comment:
Do we want to also link the version skew policy from the manual and automated upgrade pages? We get a fair bit of questions about skipping versions on Users Slack.

Reply:
Good idea